coralproject/talk
# Add localization for QuestionBoxBuilder

Hi,

## What does this PR do?

- Introduces a new localization key: `configure.include_an_icon' to allow localization of the QuestionBoxBuilder Title (Include an Icon)
- Adds propTypes to QuestionBoxBuilder

## How do I test this PR?

- run talk
- go to /dev
- check the configuration tab/question box builder

I hope I'm doing this right 😄 

Kind regards
Christian
